### ISSUE TITLE
Code manager APIs contain full license and release notes text

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#13651]
The version browse and compare API contain license information and release notes. In some cases it can be quite large, take the following request:
https://addons.mozilla.org/api/v5/reviewers/addon/920304/versions/4957090/?lang=en-US
Size: 90k
Size gzipped: 20k

Without license information:
Size: 50k
Size gzipped: 6k

Does the code manager need that info ? If we were to drop both both release notes and license information, not only would the payload be reduced in size, but we would also avoid a database query or two (There might be more things we could look into dropping, like compatibility info, which doesn't increase payload but does add database queries)

**Comment by bobsilverberg** [comment]
We don't need either of those, nor do we need compatibility info.

**Comment by diox** [comment]
Ok, it seems present in a lot of requests, so I think we could get interesting perf wins for some add-ons that do contain license/release notes.

**Comment by diox** [comment]
So, the browse, compare and draft comment API all deal with versions, and all have the following fields for a version:
* `id`
* `channel`
* `compatibility`
* `edit_url`
* `is_strict_compatibility_enabled`
* `license`
* `release_notes`
* `reviewed`
* `version`

I'm thinking of stripping that down to:
* `id`
* `channel`
* `reviewed`
* `version`

For all 3 APIs. Is that ok @bobsilverberg ?

**Comment by bobsilverberg** [comment]
Yes, that looks fine @diox. We should update our types to match that. I will open a code manager issue for that.

**Comment by bobsilverberg** [comment]
I have confirmed that we do not make use of the following fields from the version response:

* `compatibility`
* `edit_url`
* `is_strict_compatibility_enabled`
* `license`
* `release_notes`

We should update our types after this change lands, but it can land without having any impact on the operation of code-manager.

**Comment by ioanarusiczki** [comment]
@diox 
- On dev I made requests with Postman, on Browse, Compare, Version and Draft Comments endpoints and the following fields[ mentioned ](https://github.com/mozilla/addons-server/issues/13651#issuecomment-595788486) have been removed.
- the Browse, Compare, Version and Draft Comments endpoints display in the response "id", "channel", "reviewed" and "version".   
- all the above apply in the response of the POST draft comment endpoint. 

![API](https://user-images.githubusercontent.com/33448286/76216743-cd856f80-6219-11ea-9123-146bf91d16fe.png)
